velox/expression/fuzzer/ExpressionFuzzer.cpp

```diff
@@ -435,16 +436,20 @@ bool useTypeName(
 
 bool isSupportedSignature(
     const exec::FunctionSignature& signature,
-    bool enableComplexType) {
-  // Not supporting lambda functions, or functions using decimal and
-  // timestamp with time zone types.
+    bool enableComplexType,
+    bool enableDecimalType) {
+  // When enableComplexType is disabled, not supporting complex functions.
+  const bool useComplexType = useTypeName(signature, "array") ||
+      useTypeName(signature, "map") || useTypeName(signature, "row");
+  // When enableDecimalType is disabled, not supporting decimal functions. Not
```

Comment:
> Not supporting lambda functions

this comment doesn't seem right; fuzzer supports lambda functions

Reply:
I see. Updated as below.

> Not supporting functions using custom types, timestamp with time zone types and interval day to second types.